tools/CustomTasks/StripNamespaces.cs

```diff
@@ -16,7 +16,7 @@ class StripNamespaces
             { "RS4", 6 },
             { "RS5", 7 },
             { "19H1", 8},
-            { "21H1", 13}
+            { "21H1", 14}
```

Comment:
Do ListViewItem work as expect in 21h1? microsoft-ui-xaml\dev\CommonStyles\ListViewItem_themeresources_21h1.xaml is marked with 21h1 because Is21H1OrHigher is actually 21h2 now